src/Compilers/CSharp/Test/Syntax/Parsing/RecordParsing.cs

```diff
@@ -3145,11 +3145,56 @@ public void RecordRecordParsing()
             EOF();
         }
 
-        [Fact, CompilerTrait(CompilerFeature.RecordStructs)]
-        public void RecordStructParsing_WrongOrder()
+        [Fact, CompilerTrait(CompilerFeature.RecordStructs), WorkItem(59534, "https://github.com/dotnet/roslyn/issues/59534")]
+        public void RecordStructParsing_WrongOrder_CSharp10()
```

Comment:
Consider making these theories over `struct` vs `class`.

Reply:
I think we have sufficient coverage for class case now (added a few tests)